docs/reference/mount-write-mode-semantics.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Mount Write Mode Semantics
+description: Everest mount file system semantics, limitations, and and general info.
+grand_parent: Features
+parent: Mount
+---
+# Mount Write Mode Semantics
+
+{: .d-inline-block }
+experimental
+{: .label .label-red }
+
+## Consistency Model
+
+### File System Consistency
+Everest mount provides a strong read-after-write consistency model. 
+This means that once a write operation is done, the data is guaranteed to be available for subsequent read operations.
+
+### lakeFS Consistency
+Local changes are reflected in lakeFS only after the changes are **committed**.
+That means that the data is not available for other users until the changes are committed.
+If, for example, two users mount the same branch, they will not see each other's changes until they are committed.
+
+## Mount Write Mode File System Behavior
+
+### Functionality Limitations
+- Newly created empty directories will not reflect as directory marker in lakeFS.
+- LakeFS allows having 2 path keys that one is a "directory" prefix of the other, for example the following 2 lakeFS keys are valid: `animals/cat.png` and `animals` (empty object) but since a file system cannot contain both a file and a directory of the same name it will lead to an undefined behavior depending on the Filesystem type (e.g., dir and dir/file).
```

Comment:
LakeFS?
Shame Shame Shame
```suggestion
- lakeFS allows having 2 path keys that one is a "directory" prefix of the other, for example the following 2 lakeFS keys are valid: `animals/cat.png` and `animals` (empty object) but since a file system cannot contain both a file and a directory of the same name it will lead to an undefined behavior depending on the Filesystem type (e.g., dir and dir/file).
```